packages/client/src/helpers/urls.ts

```diff
@@ -15,6 +15,9 @@ export const endpointUrlWithParams = (
     return `${env("APP_URL")}${endpointUrl}?${formattedParams}&${VERSION_ID_PARAM}=${env("VERSION_ID")}`;
 };
 
+export const cdnUrl = (url: string) =>
+    `${import.meta.env.DEV ? "" : env("CDN_URL")}${url}`;
```

Comment:
Trodde først denne returnerte en tom string i dev 😅 

Kanskje dette er mer lesbart?
```suggestion
    import.meta.env.DEV ? url : `${env("CDN_URL")}${url}`;
```

Reply:
Evt sett en tom CDN_URL i client_env i dev mode, så vi slipper denne sjekken.